assets/stylesheets/mobile/chat-message.scss

```diff
@@ -0,0 +1,141 @@
+.catch-click > * {
+  pointer-events: none;
+}
+
+.chat-message-selected {
+  .chat-msgactions-hover {
+    bottom: 0;
+    transition: top 1s linear;
+  }
+}
+
+.chat-msgactions-backdrop {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: calc(100% - var(--header-offset));
+  width: 100%;
+  z-index: z("usercard");
+  transition: background-color 0.2s ease;
+
+  &.fade-in {
+    background-color: rgba(0, 0, 0, 0.75);
```

Comment:
This is a personal preference, but I'd like to try with 0.25 or 0.35 opacity. The background darkening feels way too strong, especially since the header is excluded. 

In fact, curious if we can try some other funky things here, like a bit of blurring maybe? (I am aware we are diverging from our norms, but chat feels like a good place to try new things.)

Reply:
Im not a fan of blurry background, and I would want to avoid future discussions on the PR to be about this, but I guess we can try in the future.

I did change to 0.35 though, seems better I agreed :+1: